sdk/python/foundation-models/system/finetune/video-multi-object-tracking/mmtracking-video-multi-object-tracking.ipynb

```diff
@@ -0,0 +1,1177 @@
+   "outputs": [],
+   "source": [
+    "from azure.ai.ml.entities import AmlCompute\n",
+    "from azure.core.exceptions import ResourceNotFoundError\n",
+    "\n",
+    "model_import_cluster_name = \"sample-model-import-cluster\"\n",
+    "try:\n",
+    "    _ = workspace_ml_client.compute.get(model_import_cluster_name)\n",
+    "    print(\"Found existing compute target.\")\n",
+    "except ResourceNotFoundError:\n",
+    "    print(\"Creating a new compute target...\")\n",
+    "    compute_config = AmlCompute(\n",
+    "        name=model_import_cluster_name,\n",
+    "        type=\"amlcompute\",\n",
+    "        size=\"Standard_D12_v2\",\n",
+    "        idle_time_before_scale_down=120,\n",
+    "        min_instances=0,\n",
+    "        max_instances=4,\n",
+    "    )\n",
+    "    workspace_ml_client.begin_create_or_update(compute_config).result()"
+   ]
+  },
+  {
+   "attachments": {},
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "##### Create GPU compute for finetune component\n",
+    "\n",
+    "The list of GPU machines can be found [here](https://learn.microsoft.com/en-us/azure/virtual-machines/sizes-gpu)."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "finetune_cluster_name = \"sample-finetune-cluster-gpu\"\n",
+    "\n",
+    "try:\n",
+    "    _ = workspace_ml_client.compute.get(finetune_cluster_name)\n",
+    "    print(\"Found existing compute target.\")\n",
+    "except ResourceNotFoundError:\n",
+    "    print(\"Creating a new compute target...\")\n",
+    "    compute_config = AmlCompute(\n",
+    "        name=finetune_cluster_name,\n",
+    "        type=\"amlcompute\",\n",
+    "        size=\"STANDARD_NC6s_v3\",\n",
+    "        idle_time_before_scale_down=120,\n",
+    "        min_instances=0,\n",
+    "        max_instances=4,\n",
+    "    )\n",
+    "    workspace_ml_client.begin_create_or_update(compute_config).result()"
+   ]
+  },
+  {
+   "attachments": {},
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### 3. Pick a foundation model to fine tune\n",
+    "\n",
+    "We will use the `bytetrack-yolox-x-crowdhuman-mot17-private-half` model in this notebook. If you need to fine tune a model that is available on MMTracking model zoo, but not available in `azmlft-dev-registry01` dev registry, you can either register the model and use the registered model or use the `model_name` parameter to instruct the components to pull the model directly from MMTracking model zoo.\n",
+    "\n",
+    "Currently we support tracking-by-detection models, ByteTrack and OCSort, as follows:\n",
+    "\n",
+    "| Model Name | Source |\n",
+    "| :------------: | :-------:  |\n",
+    "| [bytetrack-yolox-x-crowdhuman-mot17-private-half](https://ml.azure.com/registries/azmlft-dev-registry01/models/bytetrack-yolox-x-crowdhuman-mot17-private-half/version/8) | azureml registry |\n",
```

Comment:
I think the model name format is different in azureml regsitries. Please update all the references in these examples: bytetrack_yolox_x_crowdhuman_mot17-private-half